mapo/agents/mapo/off_mapo.py

```diff
@@ -14,6 +14,17 @@
 DEFAULT_CONFIG = merge_dicts(
     BASE_CONFIG,
     {
+        # === MAPO ===
+        # How many samples to draw from the dynamics model
+        "branching_factor": 1,
+        # Whether to use the env's dynamics to calculate the actor loss
+        "use_true_dynamics": False,
+        # Which model-learning optimization to use
+        # Valid values: "mle" (Maximum Likelihood Estimation),
+        # "pga" (DPG-aware loss function),
+        "model_loss": "mle",
+        # Which kernel metric to compare gradients in dynamics loss
+        "kernel": "l2",
```

Comment:
We don't need to repeat configs declared under `mapo.agents.mapo.mapo.DEFAULT_CONFIG`, as `merge_dict` above makes sure both on- and -off-policy versions share configs. 

IMO, the only configs that should be declared here are ones not present in the on-policy version (such as exploration strategies) or overwrites for hyperparameters which we find reasonable after testing.

Reply:
ok. noted! gonna change that.